.github/workflows/build.yml

```diff
@@ -17,17 +17,13 @@ jobs:
 
     steps:
     - name: Select Xcode
-      run: sudo xcode-select -switch /Applications/Xcode_12.2.app
+      run: sudo xcode-select -switch /Applications/Xcode_12.4.app
     - name: Xcode version
       run: /usr/bin/xcodebuild -version    
     - uses: actions/checkout@v2
     - name: Build and Test Proton
       run: xcodebuild build test -destination "name=iPhone 11 Pro" -scheme "Proton"
     - name: Capture Proton Code Coverage
       uses: codecov/codecov-action@v1.0.7
-    - name: Build and Test Proton Extensions
```

Comment:
ProtonExtension has been dropped as it was only incurring maintenance overhead with no real benefit added.